packages/patternfly-4/react-core/src/components/Page/PageHeader.js

```diff
@@ -56,42 +57,51 @@ const PageHeader = ({
   'aria-label': ariaLabel,
   ...props
 }) => (
-  <header role="banner" className={css(styles.pageHeader, className)} {...props}>
-    {(showNavToggle || logo) && (
-      <div className={css(styles.pageHeaderBrand)}>
-        {showNavToggle && (
-          <div className={css(styles.pageHeaderBrandToggle)}>
-            <Button
-              id="nav-toggle"
-              onClick={onNavToggle}
-              aria-label={ariaLabel}
-              aria-controls="page-sidebar"
-              aria-expanded={isNavOpen ? 'true' : 'false'}
-              variant={ButtonVariant.plain}
-            >
-              <BarsIcon />
-            </Button>
-          </div>
-        )}
-        {logo && (
-          <a className={css(styles.pageHeaderBrandLink)} {...logoProps}>
-            {logo}
-          </a>
-        )}
-      </div>
-    )}
-    {/* Hide for now until we have the context selector component */}
-    {/* <div className={css(styles.pageHeaderSelector)}>
-      pf-c-context-selector
-    </div> */}
-    {topNav && <div className={css(styles.pageHeaderNav)}>{topNav}</div>}
-    {(toolbar || avatar) && (
-      <div className={css(styles.pageHeaderTools)}>
-        {toolbar}
-        {avatar}
-      </div>
-    )}
-  </header>
+  <PageContextConsumer>
+    {({ isManagedSidebar, onNavToggle: managedOnNavToggle, isNavOpen: managedIsNavOpen }) => {
+      const navToggle = isManagedSidebar ? managedOnNavToggle : onNavToggle;
+      const navOpen = isManagedSidebar ? managedIsNavOpen : isNavOpen;
+
+      return (
+        <header role="banner" className={css(styles.pageHeader, className)} {...props}>
+          {(showNavToggle || logo) && (
+            <div className={css(styles.pageHeaderBrand)}>
+              {showNavToggle && (
+                <div className={css(styles.pageHeaderBrandToggle)}>
+                  <Button
+                    id="nav-toggle"
+                    onClick={navToggle}
+                    aria-label={ariaLabel}
+                    aria-controls="page-sidebar"
+                    aria-expanded={navOpen ? 'true' : 'false'}
+                    variant={ButtonVariant.plain}
+                  >
+                    <BarsIcon />
+                  </Button>
+                </div>
+              )}
+              {logo && (
+                <a className={css(styles.pageHeaderBrandLink)} {...logoProps}>
+                  {logo}
+                </a>
+              )}
+            </div>
+          )}
+          {/* Hide for now until we have the context selector component */}
```

Comment:
We have . context Selector.  Why are we hiding this?

Reply:
This has been here since before the context selector got created. Perhaps we should update the page component now to support it? Should be in a separate PR though. @rachael-phillips

Reply:
agreed.  We should do tis in a separate PR. @rachael-phillips we can open a separate issue to track that work.